### PR TITLE
Portfolio details popup window

### PR DESCRIPTION
**Pull request by anasbassi**
In this project, the work section of the portfolio page was generated using javaScript objects and also a popup window, new CSS classes were added to style it for both desktop and mobile versions.

### CODE REVIEW

File: popup.js [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+const headerWrapper = document.querySelector('.header');
+
+const works = [
+  {
+    name: 'Tonic',
+    jobDescription: {
+      company: 'CANOPY',
+      role: 'Back End Dev',
+      year: '2015',
+    },
```
Comment (tufoinnkuo10) [inline popup.js:11]: - [ ] please fix the up-down scroll in the mobile popup, make it full height so it looks like the Figma version. 
Screenshot (162)
figma
Screenshot (158)

File: popup.js [hunk 2]
```diff
@@ -0,0 +1,204 @@
...
+    description: {
+      featured:
+        'A daily selection of privately personalized reads; no accounts or sign-ups required.',
```
Comment (tufoinnkuo10) [inline popup.js:14]: - [ optional] you can add the size of your desktop popup a little to make things more vissible.

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
Hi @anasbassi 

There are some issues that you still need to work on to go to the next project but you are almost there!

To Highlight
✅ You used the correct Github flow
✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally
✅ popup works well
✅ gotten from array 

Required Changes ♻️
Check the comments under the review
Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Review by ShahierNashaat — APPROVED** [review]
Hi @anasbassi and @hislordshipprof , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Approved :ok_hand: :ok_hand: :ok_hand:

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**